master/pkg/aproto/master_message.go

```diff
@@ -113,47 +110,39 @@ func (c ContainerStarted) Addresses() []cproto.Address {
 		if info.NetworkSettings == nil {
 			return nil
 		}
-		networks := info.NetworkSettings.Networks
-		ipAddresses := make([]string, 0, len(networks))
-		for _, network := range networks {
-			ipAddresses = append(ipAddresses, network.IPAddress)
+
+		// We used to return a cproto.Address for each network in info.NetworkSettings.Networks that existed, but the
+		// harness only wants the tuple (host_ip, host_port) and disregards container_ip, and (!!) if it does receive
+		// two cproto.Address mappings for the rendezvous port of a container, it would just explode anyway because it
+		// expects a single address per container and doesn't know which one to use.
+		containerIPOnNetwork := "missing"
+		if containerNetwork := info.NetworkSettings.Networks[networkMode.NetworkName()]; containerNetwork != nil {
```

Comment:
also can we delete containerIP eventually? I know you were implying it was unused, I haven't looked at usages

Reply:
yeah it's unused.